### DISCUSSION TITLE
Re-subscribing to the copilot causes multiple times charges

### DISCUSSION THREAD

**Discussion by dengyusys** [community/community#128005] [Copilot Conversations]
### Select Topic Area

Question

### Body

# 中文内容如下

因为我的银行卡信息过期的问题，在今年6月2日扣款的时候，没有正常支付github copilot的费用，于是系统提示将于一周后再次扣款。我担心会未付款的情况会影响正常使用，于是我立即取消了github copilot的订阅，重新订阅了一次，于6月2日付款100美元。

在6月9日，我收到了一次付款提示的，我的银行卡被扣去了一年的费用。这让我非常困惑，我试图通过退款的方式解决这个问题，但是退款之后，我github copilot也失效了。

最后我损失了100美元，并且我的github copilot也无法使用了。痛哭！！！我现在在等待工单处理

希望各位网友不要像我一样重新订阅，如果扣款失败了，请耐心等待几天

# The following is an English translation

Due to the expiration of my bank card information, I did not normally pay the github copilot fee when I made the deduction on June 2 this year, so the system indicated that I would make the deduction again a week later. I was worried that the non-payment situation would affect normal use, so I immediately canceled my github copilot subscription and re-subscribed, paying $100 on June 2.

On June 9, I received a payment reminder that my bank card was charged for one year. This was very confusing to me and I tried to fix the problem with a refund, but after the refund, my github copilot also failed.

I ended up losing $100 and my github copilot stopped working. Cry!    I'm waiting for the work order to be processed

I hope you don't re-subscribe like I did, and if the deduction fails, please be patient for a few days。

![Payment history](https://github.com/community/community/assets/53852727/8270ffed-192f-45ba-9b12-3ca3c12bdf89)

**Comment by dengyusys** [comment]
工单已处理

The work order has been processed

**Comment by xiruier** [comment]
在个人订阅Github Copilot时连续被扣了两次费用，如何处理呢。

**Reply by dengyusys** [reply]
今天才看到你的消息，希望你的问题已经解决了

**Comment by Silvain-Hsiao** [comment]
嗨，我也遇到了相同的問題。

已經發出工單 (#3062777)，但至今為止已經過了五天沒有任何人協助處理我的問題。

我正在尋找任何可行的方式，試圖聯繫到 GitHub Support 人員 (希望有…)

**Reply by dengyusys** [reply]
可能是处理的比较慢吧，我也等了四到五天的样子
